src/app/core/mocks/grammar.json

```diff
@@ -0,0 +1,6 @@
+[
```

Comment:
Why you create empty array?

Reply:
I didn't finish this part because I want to complete the questions model first.